vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/core/LayoutableShadowNode.cpp

```diff
@@ -279,8 +279,10 @@ ShadowNode::Shared LayoutableShadowNode::findNodeAtPoint(
     auto centerY =
         transformedFrame.origin.y + transformedFrame.size.height / 2.0;
 
+    // [Windows - #13644
```

Comment:
This looks odd? The variables are used down below - shouldn't those also be in a `// [[Windows` block?

Reply:
So this file was forked last integration, but the upstream PR wasn't quite correct. I created a new PR that fixes it and also reverts the previous change so that's why this is in the block. 

PR for reference:

https://github.com/facebook/react-native/pull/46447